### ISSUE TITLE
Question about downscaling

### ISSUE THREAD

**Issue by ChrisAGBlake** [IBM/granite-wxc#6]
Firstly thank you so much for developing and publishing this model and code. Much appreciated!

The downscaling inference example is using a model that was finetuned on the MERRA2 dataset to downscale 2m temp 6x. If I understand correctly the finetuning was done by generating a course version of the MERRA2 data as the input and using the full resolution as the target. Does this mean that the finetuned model can only downscale an input with the same lat lon resolution that it was trained with as the input, or could it be applied to the full resolution MERRA2 data to further downscale another 6x? I assume the former but just wanted to check.

**Comment by johannesschmude** [comment]
You're correct. This is supervised downscaling so you need some form of ground truth. The weights that we released with this code (https://huggingface.co/ibm-granite/granite-geospatial-wxc-downscaling) stem from coarsening MERRA-2 by 6x _and_ applying a 3x3 smoothing filter. So as such, the weights we released are only applicable to this problem.

Having said that, we use the same code that is released here for our CORDEX downscaling work [documented in the paper](https://arxiv.org/abs/2409.13598). We have not released the weights yet though. So you can use the code here for applications beyond MERRA-2. When doing so, you will need a high resolution ground truth. Also, you might want to unfreeze part of the core model or do some sort of LoRA tuning to get best results.

**Comment by ChrisAGBlake** [comment]
Thanks very much. Would it be possible to share any code examples for downscaling over a regional area or something similar to the CORDEX downscaling you show in the paper?
